hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/state/IdpSelectedState.java

```diff
@@ -42,7 +42,15 @@ public IdpSelectedState(
             List<String> availableIdentityProviders,
             boolean transactionSupportsEidas) {
 
-        super(requestId, requestIssuerId, sessionExpiryTimestamp, assertionConsumerServiceUri, sessionId, transactionSupportsEidas);
+        super(
+            requestId,
+            requestIssuerId,
+            sessionExpiryTimestamp,
+            assertionConsumerServiceUri,
+            sessionId,
+            transactionSupportsEidas,
+            null
```

Comment:
Why `null` here?

Reply:
This isn't an end state is it?

Reply:
The null is to maintain the existing behaviour; this state never had `forceAuthentication`, and so we don't introduce it here.

I'm assuming the reason it's not included with this state is that `forceAuthentication` is consumed by the IDP, and that by this point in the journey it's been consumed and is no longer required? That's a guess though...

Reply:
I think it's fine to leave it out for now as like you said Chris is not used anywhere at the moment and it also means we don't have to make collateral changes to the tests, etc